File: Simulador De Mercado/src/main/java/ar/com/carrion/simuladordemercado/backend/Domains/Order.java
package ar.com.carrion.simuladordemercado.backend.Domains;


import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.stereotype.Component;

import java.time.LocalDateTime;

@Entity
@Table(name = "Orders")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Component
public class Order {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private double price;
    private String typeOrder;
    private int quantity;
    private LocalDateTime time;

}

File: Simulador De Mercado/src/main/java/ar/com/carrion/simuladordemercado/frontend/frontEndMarketSimulator/src/app/Feature/order/order.ts
import { Component } from '@angular/core';

@Component({
  selector: 'app-order',
  imports: [],
  templateUrl: './order.html',
  styleUrl: './order.scss',
})
export class Order {
  price = 0.00;
  typeOrder = "";
  quantity = 0;
  time = "";
}
